pkg/dogstatsd/parse_events_test.go

```diff
@@ -98,6 +98,18 @@ func TestEventError(t *testing.T) {
 	_, err = parseEvent([]byte("_e:|text"))
 	assert.Error(t, err)
 
+	// invalid title length
+	_, err = parseEvent([]byte("_e{-123,-987}:"))
+	assert.Error(t, err)
+
+	// invalid text length
+	_, err = parseEvent([]byte("_e{5,-987}:title"))
+	assert.Error(t, err)
+
+	// malformed message
+	_, err = parseEvent([]byte("_e{0001,-9876"))
+	assert.Error(t, err)
+
```

Comment:
For the sake of completeness, you could add a malformed event with a 0 length title but proper text length+text: `_e{0,4}:|text`

Reply:
That's a good point - I'll add that in